ngraph/python/tests/test_ngraph/test_normalization.py

```diff
@@ -115,3 +115,32 @@ def test_batch_norm_inference():
     result = run_op_node([data, gamma, beta, mean, variance], ng.batch_norm_inference, epsilon)
 
     assert np.allclose(result, excepted)
+
+
+def test_mvn():
+    data = np.array([[[[0.8439683], [0.5665144], [0.05836735]],
+                      [[0.02916367], [0.12964272], [0.5060197]],
+                      [[0.79538304], [0.9411346], [0.9546573]]],
+                     [[[0.17730942], [0.46192095], [0.26480448]],
+                      [[0.6746842], [0.01665257], [0.62473077]],
+                      [[0.9240844], [0.9722341], [0.11965699]]],
+                     [[[0.41356155], [0.9129373], [0.59330076]],
+                      [[0.81929934], [0.7862604], [0.11799799]],
+                      [[0.69248444], [0.54119414], [0.07513223]]]], dtype=np.float32)
```

Comment:
I propose to take different input for which it will be easier to compute output manually. Now, personally for me, it is not evident why I should believe that expected result is true.